### DISCUSSION TITLE
百宝箱主页

### DISCUSSION THREAD

**Discussion by 83249287423** [Meloong-Git/PCL#4570] [自定义主页]
链接：https://gitee.com/huwis/treasure-box-homepage/releases/download/v1.0/Custom.xaml

**Comment by 83249287423** [comment]
![屏幕截图 2024-08-23 105531](https://github.com/user-attachments/assets/4c828204-39ac-4d78-95fc-a3106aa2250d)

**Comment by Deep-Dark-Forest** [comment]
与简单主页功能类似，不会添加到 PCL

**Reply by MoYuan-CN** [reply]
参见 https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-9641112 https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-10330643
另外我这里回调给了错误 Code 1010

**Reply by 83249287423** [reply]
> 参见 [#2528 （在线程中回复）](https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-9641112) [#2528 （评论）](https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-10330643) 另外我这里回调给了错误 Code 1010

已修复
